---
title: Papers

author_profile: false

permalink: /papers/
---

### 2025
+ Vieira Mourato, B and Haubold, B. (2025) [Fast detection of unique genomic regions](https://doi.org/10.1016/j.csbj.2025.02.025). Computational and Structural Biotechnology Journal, 27:843-850.

### 2024
+ Vieira Mourato, B., Tsers, I., Denker, S., Klötzl, F., and Haubold, B. (2024). [Marker discovery in the large](https://doi.org/10.1093/bioadv/vbae113). Advances in Bioinformatics, 4:vbae113.
+ Vieira Mourato, B. and Haubold, B. (2024). [Detection and Annotation of Unique Regions in Mammalian Genomes](https://doi.org/10.1093/g3journal/jkae257). G3, 15:jkae257.

### 2021
+ Haubold, B., Klötzl, F., Hellberg, L., Thompson, D., and Cavalar, M. (2021). [Fur: Find unique genomic regions for diagnostic PCR](https://doi.org/10.1093/bioinformatics/btab059). Bioinformatics. [Software](https://github.com/evolbioinf/fur).
+ Haubold, B. and Klötzl, F. (2021). [Fast phylogeny reconstruction from genomes of closely related microbes](https://link.springer.com/protocol/10.1007/978-1-0716-1099-2_6). In Bacterial Pangenomics, 2nd Ed.: Methods and Protocols. Springer, p. 77–89.

### 2020
+ Klötzl, F. and Haubold, B. (2020). [Phylonium: Fast estimation of evolutionary distances from large samples of similar genomes](https://doi.org/10.1093/bioinformatics/btz903). Bioinformatics, 36:2040–2046. [Software](https://github.com/evolbioinf/phylonium).
+ Lynch, M., Haubold, B., Pfaffelhuber, P., and Maruki, T. (2020). [Inference of historical population-size changes with allele frequency data](https://doi.org/10.1534/g3.119.400854). G3, 10:211–223. [Software](https://github.com/evolbioinf/epos).

### 2019
+ Pirogov, A., Pfaffelhuber, P., Börsch-Haubold, A., and Haubold, B. (2019). [High-complexity regions in mammalian genomes are enriched for developmental genes](https://doi.org/10.1093/bioinformatics/bty922). Bioinformatics, 35:1813–1819. [Software](https://github.com/evolbioinf/macle/).

### 2016
+ Klötzl, F. and Haubold, B. (2016). [Support values for genome phylogenies](https://doi.org/10.3390/life6010011). Life, 6:11. [Software and data sets](http://evolbioinf.github.io/life2015/).
+ Odenthal-Hesse, L., Dutheil, J. Y., Klötzl, F., and Haubold, B. (2016). [hotspot: Software to support sperm-typing for investigating recombination hotspots](https://doi.org/10.1093/bioinformatics/btw195). Bioinformatics, 32:2554–2555. [Software](https://github.com/evolbioinf/hotspot).

### 2015
+ Haubold, B., Klötzl, F., and Pfaffelhuber, P. (2015). [andi: Fast and accurate estimation of evolutionary distances between closely related genomes](https://doi.org/10.1093/bioinformatics/btu815). Bioinformatics, 31:1169–1175. [Software](https://github.com/evolbioinf/andi/).
+ Krause, L., Haubold, B., and Börsch-Haubold, A. G. (2015). [Social exclusion changes histone modifications H3K4me3 and H3K27ac in liver tissue of wild house mice](https://doi.org/10.1371/journal.pone.0133988). PLOSone, DOI: 10.1371/journal.pone.0133988.

### 2014
+ Börsch-Haubold, A. G., Montero, I., Konrad, K., and Haubold, B. (2014). [Genome-wide quantitative analysis of histone H3 lysine 4 trimethylation in wild house mouse liver: Environmental change causes epigenetic plasticity](https://doi.org/10.1371/journal.pone.0097568). PLOSone, DOI: 10.1371/journal.pone.0097568. Press Release.
+ Lynch, M., Xu, S., Maruki, T., Jiang, X., Pfaffelhuber, P., and Haubold, B. (2014). [Genome-wide linkage-disequilibrium profiles from single individuals](https://doi.org/10.1534/genetics.114.166843). Genetics, 198:269–281.

### 2013
+ McCann, H. C., Rikkerink, E. H. A., Bertels, F., Fiers, M., Lu, A., Rees-George, J., Andersen, M. T., Gleave, A. P., Haubold, B., Wohlers, M. W., Guttman, D. S., Wang, P. W., Straub, C., Vanneste, J. L., Rainey, P. B., and Templeton, M. D. (2013). [Genomic Analysis of the Kiwifruit Pathogen Pseudomonas syringae pv. actinidiae Provides Insight into the Origins of an Emergent Plant Disease](https://doi.org/10.1371/journal.ppat.1003503). PLOS-Pathogens, 9:e1003503.
+ Thota, A., Haubold, B., Michael, S, Doak,T., Xu, S., and Henschel, R. (2013). [Making campus bridging work for researchers: a case study with mlRho](https://doi.org/10.1145/2484762.2484803). In: Proceedings of the Conference on Extreme Science and Engineering Discovery Environment: Gateway to Discovery, 17:1–17:8. ACM, New York.
+ Haubold, B., Krause, L., Horn, T., and Pfaffelhuber, P. (2013). [An Alignment-free Test for Recombination](https://doi.org/10.1093/bioinformatics/btt550). Bioinformatics, 24:3121–3127.

### 2012
+ Huang, W., Haubold, B., Hauert, C. and Traulsen, A. (2012). [Emergence of stable polymorphisms driven by evolutionary games between mutants](https://doi.org/10.1038/ncomms1930). Nature Communications, 3:919.
+ Haubold, B. and Pfaffelhuber, P. (2012). [Alignment-Free Population Genomics: An Efficient Estimator of Sequence Diversity](https://doi.org/10.1534/g3.112.002527). Genes, Genomes, Genetics, 2:883–889. [Software](http://guanine.evolbio.mpg.de/pid/).

### 2011
+ Haubold, B., Reed, F. and Pfaffelhuber, P. (2011). [Alignment-free estimation of nucleotide diversity](https://doi.org/10.1093/bioinformatics/btq689). Bioinformatics, 27:449–455. [Software](http://guanine.evolbio.mpg.de/pim/).
+ Domazet-Lošo, M. and Haubold, B. (2011). [Alignment-free detection of local similarity among viral and bacterial genomes](https://doi.org/10.1093/bioinformatics/btr176). Bioinformatics, 27:1466–1472. [Software](http://guanine.evolbio.mpg.de/alfy/).
+ Tellier, A., Pfaffelhuber, P., Haubold, B., Naduvilezhath, L., Rose, L. E., Städler, T., Stephan, W. and Metzler, D. (2011). [Estimating parameters of speciation models based on refined summaries of the joint site-frequency spectrum](https://doi.org/10.1371/journal.pone.0018155). PLoSone, 6:18155.
+ Domazet-Lošo, M. and Haubold, B. (2011). [Alignment-free detection of horizontal gene transfer between closely related bacterial genomes](https://doi.org/10.4161/mge.1.3.18065). Mobile Genetic Elements, 1:230–235.

### 2010
+ Wolf, J., Bayer, T., Haubold, B., Schilhabel, M., Rosenstiel, P. and Tautz, D. (2010). [Nucleotide divergence versus gene expression differentiation: 454 transcriptome sequencing in natural isolates from the carrion crow and its hybrid zone with the hooded crow](https://doi.org/10.1111/j.1365-294x.2009.04471.x). Molecular Ecology, 19:162–175.
+ Haubold, B., Pfaffelhuber, P. and Lynch, M. (2010). [mlRho—A program for estimating the population mutation and recombination rates from shotgun-sequenced diploid genomes](https://doi.org/10.1111/j.1365-294x.2009.04482.x). Molecular Ecology, 19:277–284. [Software](http://guanine.evolbio.mpg.de/mlRho/).

### 2009
+ Städler, T., Haubold, B., Merino, C., Stephan, W., Pfaffelhuber, P. (2009). [The impact of sampling schemes on the site frequency spectrum in non-equilibrium subdivided populations](https://doi.org/10.1534/genetics.108.094904). Genetics, 182:205–216.
+ Haubold, B., Pfaffelhuber, P., Domazet-Lošo, M. and Wiehe, T. (2009). [Estimating mutation distances from unaligned genomes](https://doi.org/10.1089/cmb.2009.0106). Journal of Computational Biology, 16: 1487–1500. [Software](http://guanine.evolbio.mpg.de/cgi-bin/kr/kr.cgi.pl).
+ Domazet-Lošo, M. and Haubold, B. (2009). Efficient estimation of pairwise distances between genomes. Bioinformatics, 25: 3221–3227. [Software](http://guanine.evolbio.mpg.de/cgi-bin/kr2/kr.cgi.pl).

### 2008
+ Haubold, B., Domazet-Lošo, M. and Wiehe, T. (2008). [An alignment-free distance measure for closely related genomes](https://link.springer.com/chapter/10.1007/978-3-540-87989-3_7). In C. R. Nelson and S. Vialette, editors, RECOMB-CG 2008, volume 5267 of Lecture Notes in Bioinformatics, 87–99. Springer.
